index.js

```diff
@@ -544,26 +545,23 @@ Command.prototype.executeSubCommand = function(argv, args, unknown) {
   if (exists(localBin + '.js')) {
     bin = localBin + '.js';
     isExplicitJS = true;
-  } else if (exists(localBin)) {
+  } else if (process.platform === 'win32' ? exists(localBin + '.cmd') : exists(localBin)) {
```

Comment:
Can you please put it to own variable, for readability purpose

Reply:
Sure thing.